.github/workflows/deploy.yaml

```diff
@@ -20,14 +19,13 @@ jobs:
       - name: Checkout Docs
         run: git fetch && git checkout ${GITHUB_SHA}
         working-directory: ./product/coder.com/site/docs
-      - name: Setup Node
-        uses: actions/setup-node@v1
-      - name: Deploy
-        run: ./product/coder.com/site/docs/.github/ci/deploy.sh
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-          GITHUB_REF: ${{ github.ref }}
-          GITHUB_REPOSITORY: ${{ github.repository }}
-          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
-          VERCEL_ORG_ID: team_tGkWfhEGGelkkqUUm9nXq17r
-          VERCEL_PROJECT_ID: QmZRucMRh3GFk1817ZgXjRVuw5fhTspHPHKct3JNQDEPGd
+      - name: Commit files
+          run: |
+            git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+            git config --local user.name "github-actions[bot]"
+            git commit -m "cdr/docs: bump product/coder.com/site/docs ${GITHUB_SHA}" -a
```

Comment:
We can definitely change these. I was following https://github.com/marketplace/actions/github-push